pkg/scanners/terraform/options.go

```diff
@@ -193,10 +193,10 @@ func ScannerWithDownloadsAllowed(allowed bool) options.ScannerOption {
 	}
 }
 
-func ScannerWithEmbeddedLibraries(embedded bool) options.ScannerOption {
```

Comment:
This option was not used because there is a global option for using embedded libraries.  https://github.com/aquasecurity/defsec/blob/master/pkg/scanners/options/scanner.go#L62